src/WidgetMessaging.js

```diff
@@ -96,6 +96,45 @@ export default class WidgetMessaging {
             });
     }
 
+    /**
+     * Toggle Jitsi Audio Mute
+     * @return {Promise} To be resolved when action completed
+     */
+    toggleJitsiAudio() {
+        return this.messageToWidget({
+            api: OUTBOUND_API_NAME,
+            action: "audioToggle",
```

Comment:
these 3 actions should require a bump in API version, as shown here: https://github.com/matrix-org/matrix-react-sdk/pull/2781/files#diff-48acd843c021678a317564b0cf6d8648R24

Also, if you need to figure out if the widget will even support it, a toWidget API request for `supported_api_versions` should reveal that.

Reply:
Sending to an older version is alright though, as it will just ignore the command?